string path
string dest
---
bool resp
